changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1 | 2020-03-23
+* Drop matplotlib < 3 requirement.
+* Add functionality which redirects users to API when accessing particular fields.
+* Add API for markers.
+* Bugfix CorrelatedStack.plot().
```

Comment:
What did the bug prevent people from doing? If someone had this bug and they went to the changelog, they should be able to recognize that it's been fixed.

Reply:
* Bugfix CorrelatedStack.plot() which resulted in the function throwing an error rather than showing a single frame.

This ties in with a bigger issue, which is that it would be nice to at least test whether graphical functions work without throwing. This would prevent stuff like this. I will investigate this, but don't really want to do this as part of this PR.